docker-compose.yaml

```diff
@@ -1,19 +1,20 @@
-version: '3.9'
+version: "3.3"
+
 services:
   website:
-    build:
-      context: .
-      dockerfile: ./website/Dockerfile
-      args:
-        - HUGO_BASEURL=https://perses.dev
+    image: hugomods/hugo
+    command: hugo server --watch --bind=0.0.0.0 --disableLiveReload=true --minify -environment production --baseURL https://perses.dev --appendPort=false
     restart: unless-stopped
     ports:
-      - "5001:80"
+      - 1313:1313
+    volumes:
+      - ./website:/src
```

Comment:
does that mean we need to run on the server the script `synchronize.sh` ? And if so we should probably add to ignore file every file generated if it's not done already

Reply:
I'll double check it, but I guess I already ignore it.

Reply:
based on what you did directly on the server, it looks like it's not ignored : 

```bash
$ git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   docker-compose.yaml
	modified:   scripts/synchronize.sh

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	docker-compose.old
	package-lock.json
	website/.hugo_build.lock
	website/content/docs/perses/
	website/content/docs/prologue/
	website/resources/_gen/assets/
	website/static/images/
```

Reply:
@Nexucis added can you check?